SU-SWS/decanter
# 216 Create site search component

# READY FOR REVIEW

# Summary
- Create the global site search component

# Needed By (Date)
- End of week

# Steps to Test

1. Pull this branch
1. Run grunt styleguide
2. Check that the search component matches the design in the Figma file "Stanford Design Library" -> "Header, Footer & Navigation" section. Check that the twig file is properly set up and that basic accessibility requirements are met.

# Affected Projects or Products
- Decanter

# Associated Issues and/or People
- https://github.com/SU-SWS/decanter/issues/216
- https://github.com/SU-SWS/decanter/issues/222
- For xs to md break points according to Figma documentation, this component lives within the main nav component, and is hidden from view until the main nav is expanded.
